postgres/init/22-portal-content.sql

```diff
@@ -976,6 +960,8 @@ ac1cf001-7fbc-1f2f-817f-bce0573f0009	de	This is only a test text	This is only a
 5cf74ef8-e0b7-4984-a872-474828beb510	de	The value added service solution Country Risk Score gives different users additional information regarding their business partners. This is part of the Know Your Business Partner approach and enables the user to make better decisions based on aggregated information. The main part of the country risk dashboard are information per country regarding corruption, political stability, economic risk and social and structural figures. There are official sources connected like CPI or Basel but the service offers the possibility to upload company/user specific ratings.	The country risk dashboard - country information regarding corruption, political stability, economic risk and social and structural figures.
 5cf74ef8-e0b7-4984-a872-474828beb510	en	The value added service solution Country Risk Score gives different users additional information regarding their business partners. This is part of the Know Your Business Partner approach and enables the user to make better decisions based on aggregated information. The main part of the country risk dashboard are information per country regarding corruption, political stability, economic risk and social and structural figures. There are official sources connected like CPI or Basel but the service offers the possibility to upload company/user specific ratings.	The country risk dashboard - country information regarding corruption, political stability, economic risk and social and structural figures.
+a16e73b9-5277-4b69-9f8d-3b227495dfea	en	In order to allow Data Providers and Data Consumers to easily participate in relevant Use Cases, a service for low-effort data sharing was needed. The service would add a convenience layer around already established CX components, such as EDC and Digital Twin Registry, simplifying their use. It is a standalone service which can be self-hosted, it enables companies to provide their data in the Catena-X network via an EDC, Ddata is uploaded via CSV-files/Manual Entry/Json format, the DFT registers the data in the Digital Twin Registry and makes it accessible via an EDC, DFT also act as a Data Consumer, it is show the data offers from data provider.	\N
```

Comment:
please add short-description. column description_short is not nullable

Reply:
:) done, thanks!